mail_tracking/controllers/mailbox.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo.http import request, route
+
+from odoo.addons.mail.controllers.mailbox import MailboxController
+
+
+class MailTrackingMailBoxController(MailboxController):
+    @route("/mail/failed/messages", methods=["POST"], type="json", auth="user")
+    def discuss_failed_messages(
+        self, search_term=None, before=None, after=None, limit=30, around=None
+    ):
+        """Fetch failed massages for discuss"""
```

Comment:
What you were thinking in? :stuck_out_tongue: 

```suggestion
        """Fetch failed messages for discuss"""
```